### PR TITLE
Install kubevirtci from different organization

### PR DESCRIPTION
**Pull request by qinqon**
Also move to a script the code to install it.

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: hack/install-kubevirtci.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+script_dir=$(dirname "$(readlink -f "$0")")
+kubevirtci_dir=kubevirtci
+
+rm -rf $kubevirtci_dir
+git clone https://github.com/$organization/kubevirtci $kubevirtci_dir
+pushd $kubevirtci_dir
+git checkout $commit
+popd
+
```
Comment (phoracek) [inline hack/install-kubevirtci.sh]: please drop those trailing lines

File: Makefile [hunk 2]
```diff
@@ -80,10 +81,8 @@ $(local_handler_manifest): deploy/operator.yaml
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)#" \
 		deploy/operator.yaml > $@
 
-$(CLUSTER_DIR)/%: kubevirtci.version
-	rm -rf kubevirtci
-	git clone https://github.com/kubevirt/kubevirtci
-	cd kubevirtci && git checkout $$(cat  ../kubevirtci.version)
+$(CLUSTER_DIR)/%: $(install_kubevirtci)
```
Comment (phoracek) [inline Makefile]: this should depend on kubevirtci as well, in case it is missing

File: test/e2e/utils.go [hunk 3]
```diff
@@ -172,20 +173,21 @@ func run(node string, command ...string) {
...
+	GinkgoWriter.Write([]byte(strings.Join(ssh_command, " ") + "\n"))
 	cmd.Stdout = GinkgoWriter
 	cmd.Stderr = GinkgoWriter
 	Expect(cmd.Run()).To(Succeed())
 }
 
 func createDummy(nodes []string, dummyName string) {
 	for _, node := range nodes {
-		run(node, "sudo", "ip", "link", "add", dummyName, "type", "dummy")
+		run(node, "sudo", "nmcli", "con", "add", "type", "dummy", "con-name", dummyName, "ifname", dummyName)
```
Comment (phoracek) [inline test/e2e/utils.go]: how is this related to the PR?

Comment (qinqon) [reply test/e2e/utils.go]: stdci was failing at "ip link add" with "RTNETLINK answers: File exists" but the interface was not there, looks like it's relatd to flushing the interface, doing it with nmcli works and also kind of make sense since we are testing networkmanager.

So this is all to make CI happy.

**Comment by phoracek** [comment]
Do we need this now when the PR has been merged?

**Comment by phoracek** [comment]
I like the script, but we should drop the mention of different repo. Just pin it to specific commit on kubevirtci.

**Comment by qinqon** [comment]
We are not going to be able to see commits of different branches outside of main organization, let's keep this as it is and point to the merged stuff.

We may need a long time deviation on a kubevirtci wip, this is still needed.

File: Makefile (hunk 2 above)
Comment (qinqon) [reply Makefile]: kubevirtci is part of the CLUSTER_DIR if it's not there make will call the dependencies.

CLUSTER_DIR = kubevirtci/cluster-up/

File: hack/install-kubevirtci.sh (hunk 1 above)
Comment (qinqon) [reply hack/install-kubevirtci.sh]: Done

**Comment by phoracek** [comment]
@qinqon do you want to wait for the multi-node fix?

**Comment by qinqon** [comment]
@phoracek let's merge this will fix cli.sh already and we are not testing multinode now.
